docs/customization/index.rst

```diff
@@ -49,6 +49,22 @@ For instance :
       neutron_external_interface: eth2
       tunnel_interface: eth3
 
+Running from kolla/master
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This can bite but you can run the latest Kolla code with the latest built
```

Comment:
I would write, "if you want to live on the bleeding edge, you can run ..." but it's a personal opinion.